manifests/m/Microsoft/VC++2010Redist-x64/10.0.40219.325/Microsoft.VC++2010Redist-x64.locale.en-US.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate
+
+PackageIdentifier: Microsoft.VC++2010Redist-x64
+PackageVersion: 10.0.40219.325
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft Visual C++ 2010 Service Pack 1 Redistributable Package MFC Security Update (x64)
+PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=13523

```

Comment:
```suggestion
PackageName: Microsoft Visual C++ 2010 x64 Redistributable
PackageUrl: https://www.microsoft.com/en-us/download/details.aspx?id=26999
```

Reply:
it doesn't work!

Reply:
wait

Reply:
There should be a button called add changes or something along those lines, you've just clicked "Resolved" without resolving the problem